draft-xyz-rats-psa-endorsements.md

```diff
@@ -131,94 +140,149 @@ firmware measurement associated with Implementation ID
 ~~~
 {: #ex-reference-value title="Example Reference Value"}
 
-### Firmware Updates and Patches
-{: #sec-fw-evo}
+## Identity
 
-Firmware RoT descriptors that are part of the same upgrade chain can be linked
-together using one of `comid-patches` or `comid-updates` relations, depending on
-the precise nature of their relationship.  For example, if using semantic
-versioning {{SEMA-VER}}, a bump in the MINOR version number would be associated
-with a `comid-updates` relation, whereas an increase in the PATCH indicator
-would have a corresponding `comid-patches` relation between the involved
-components.  Note that `comid-updates` relations would only occur between
-firmware RoT descriptors that have PATCH number 0 and "adjacent" MINOR numbers
-as illustrated in {{fig-updates-patches}}.  Note that changing the MAJOR
-version number would typically result in a separate product / upgrade chain.
+{: #sec-identity}
+
+Identity Claims carry the verification key associated with the Initial
+Attestation Key (IAK) of a PSA device.
+
+Each verification key is encoded alongside the corresponding device Instance ID
+in a CoMID identity claim.
+
+The Instance ID is encoded using `tagged-ueid` as `$device-id-type`.
```

Comment:
please make sure you have checked all CoMID-related stuff against the most up-to-date CDDL
```suggestion
The Instance ID is encoded using `tagged-ueid-type` as `$device-id-type-choice`.
```